Dockerfile

```diff
@@ -1,17 +1,16 @@
 FROM conanio/gcc13-ubuntu16.04
 
-COPY src /home/conan/cxx_template/src
-COPY CMakeLists.txt /home/conan/cxx_template
-COPY conanfile.py /home/conan/cxx_template
-COPY main.cxx /home/conan/cxx_template
-
-
 WORKDIR /home/conan/cxx_template
 
-RUN sudo chown -R conan /home/conan  && conan install . --output-folder=build --settings build_type=Release  --settings compiler.cppstd=gnu20 --build=missing
+COPY conanfile.py /home/conan/cxx_template
+RUN sudo chown -R conan /home/conan  \
+   && conan install . --output-folder=build --settings build_type=Release  --settings compiler.cppstd=gnu20 --build=missing
 
 WORKDIR /home/conan/cxx_template/build
+COPY src /home/conan/cxx_template/src
```

Comment:
merely moved some copies down so we don't always need to redo the conan stuff on simple edits